### DISCUSSION TITLE
AdonisJS and NuxtJS

### DISCUSSION THREAD

**Discussion by ezzaroualDotCom** [adonisjs/core#2952] [Help]
Hello guys

Anyone have any idea how to migrate the latest version of AdonisJS with NuxtJS

like this one https://github.com/jamesAtcodeninja/adonuxt

But It's very old and does not work for current version of AdonisJs

I hope anyone have some advises

Thanks in advance

**Comment by Barbapapazes** [comment]
Hello,

Can you explain more about this template? What was the objective of this template?

**Comment by McSneaky** [comment]
AFAIK there's no ready made package for it currently. You can just run Nuxt and AdonisJS as two sandalone applications. Where Nuxt provides frontend and AdonisJS acts as an API

**Comment by RomainLanz** [comment]
Hey @AlexanderTolson! 👋 

You are better off creating two separate applications than trying to bundle them together.
Use AdonisJS as your API Server and Nuxt as your Rendering Server.

Here is an example with two Nuxt application and one AdonisJS server: https://github.com/Lausanne-Esports/

**Comment by marsyeen** [comment]
Hello! There It is better to create 2 separate applications, instead what I do for one of my projects is run AdonisJS as an API but configure it to be the same route as the host frontend for example AdonisJS could be running on https://myhost.com/api and your frontend application could be running in NextJS (React) NuxtJS (Vue) fetching API data on components / pages. 

**Note : I also like to make AdonisJS a static content provider for assets, but that's up to preference.**

Hopefully this helped you 👍
